extensions/wikia/SoundCloudTag/SoundCloudTagController.class.php

```diff
@@ -38,15 +38,18 @@ public function __construct() {
 		$this->helper = new WikiaIFrameTagBuilderHelper();
```

Comment:
after all changes in WikiaParserTagController I don't see much value in extending this class by SoundCloud.
It doesn't use any of this logic